fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/bridge/traci/reader/TypeBasedTraciReader.java

```diff
@@ -71,6 +74,7 @@ protected Object readFromStream(DataInputStream in) throws IOException {
                 return readUnsignedByte(in);
             case TraciDatatypes.COMPOUND:
                 readInt(in); // required at least for reading inductionloop vehicle data compound
```

Comment:
the comment may not fit anymore

Reply:
changed to "this field needs to be read but can be ignored"